Matias/enhancement/426 add about page and designs

MatiasKanerva opened the pull request:
## 📄 **Pull Request Overview**

Closes #426 

## 🔧 **Changes Made**

Added the layout and text content from Figma's example
Added breakpoints for mobile-, tablet-, laptop and larger monitor views

---

## ✅ **Checklist Before Submission**

- **Functionality**: I have tested my code, and it works as expected. ✅
- **JSDoc**: I have added or updated JSDoc comments for all relevant code.
- **Debugging**: No `console.log()` or other debugging statements are left. ✅
- **Clean Code**: Removed commented-out or unnecessary code. ✅
- **Tests**: Added new tests or updated existing ones for the changes made.
- **Documentation**: Documentation has been updated (if applicable).

---

## 📝 **Additional Information**

Provide any additional context or information that reviewers may need to know:

Screenshots of breakpoints from the webpage:
Small phone
![image](https://github.com/user-attachments/assets/415ff48c-a6c0-4376-967e-02c6dc38d02f)

Large phone
![image](https://github.com/user-attachments/assets/e84d7c52-380d-48fd-b381-f6a31b189b65)

Tablet
![image](https://github.com/user-attachments/assets/4aad9627-66f9-4d7f-9de6-5502354a9161)

Laptop
![image](https://github.com/user-attachments/assets/2dbe2d07-bb3d-416d-bc1c-0255235da513)

Larger monitors
![image](https://github.com/user-attachments/assets/ba8ab19f-b845-4ab4-bf79-465aac4441f5)
![image](https://github.com/user-attachments/assets/6d2c50b6-89d3-43c8-993e-0302c5d680e1)

Jonroi commented:
Without seeing the code first that comes to my mind is those borders. Try to match them with figma plans:
![image](https://github.com/user-attachments/assets/a7abcee3-4a40-436e-a0bb-cd26edc5c8ed)

Jonroi reviewed (changes requested):
instructions given via discord

Jonroi commented:
how is this going? any updates? haven't got re-request.

MatiasKanerva commented:
@Jonroi I'm having slight difficulties on the placement of the year text and how it fits with the line connecting to it.
I have added the Russian translation and changed all variables on scss to global ones, I will look into adding the about page to the navbar next.

Jonroi commented:
Okay! Good to hear that it's moving forward. Thank you for replying here so everyone knows the situation. I'm little breathing to your neck that i know what to bring to next build, but no worries take your time there's no rush to finish this, well just take it to next build then.

Rutjake reviewed (approved):
Same as above, Good Work!
